packages/app/src/modules/options.js

```diff
@@ -86,6 +86,7 @@ export const options = {
     sortOrder: { defaultValue: '0', requestable: false, savable: true },
     subtitle: { defaultValue: undefined, requestable: false, savable: true },
     title: { defaultValue: undefined, requestable: false, savable: true },
+    series: { defaultValue: [], requestable: false, savable: true },
```

Comment:
Wasn't sure about how the new `options` props works, does this look correct @edoardo ?

Reply:
`requestable` is for those options that can be passed to the analytics request.
`savable` is for options that should be passed in the AO when saving it.
`defaultValue` should match the one on the backend if present.

Reply:
Thanks @edoardo , so does this look correct then? or should `series` be `requestable`?

Reply:
No, `series` is not a valid analytics api param